recipe_scrapers/__init__.py

```diff
@@ -223,13 +223,13 @@
 from .zeitwochenmarkt import ZeitWochenmarkt
 from .zenbelly import ZenBelly
 
-SCRAPERS = {
+SCRAPERS: Dict[str, Type[AbstractScraper]] = {
     ACoupleCooks.host(): ACoupleCooks,
     Abril.host(): Abril,
     AfghanKitchenRecipes.host(): AfghanKitchenRecipes,
     AkisPetretzikis.host(): AkisPetretzikis,
     AlbertHeijn.host(): AlbertHeijn,
-    AllRecipes.host(): AllRecipes,
+    AllRecipes.host(): cast(Type[AbstractScraper], AllRecipes),
```

Comment:
Hm.  I wonder whether we can simplify/refactor the `AllRecipes` scraper so that this typecast isn't necessary.

Reply:
yeah it's a little tricky because it's doing some stuff with `__new__`

I think it should be compatible because when we instantiate the type it returns an `AbstractScraper`, but I don't think mypy understands

Reply:
Ok - if it's a `mypy` misunderstanding, then I think that the next consideration is whether the typecast could affect the program's behaviour for any users.

(if so: we probably shouldn't add the typecast within this pull request; adding an `ignore` directive and raising a bug might be better, to figure out what we can do about it)

Reply:
cast doesn't actually do anything at runtime, it's a no-op -- it's only for type checkers

https://github.com/python/cpython/blob/27d8dc2c9d3de886a884f79f0621d4586c0e0f7a/Lib/typing.py#L2237-L2245